Building on Windows : CMake Error at Pangolin/cmake/EmbedShaderFiles.cmake:26

Environment: Windows 11 64 bits
Compiler: MinGW 8.1.0 64 bits

```
C:\Users\ingle\Pangolin\build>cmake .. -G "MinGW Makefiles"
-- The C compiler identification is GNU 8.1.0
-- The CXX compiler identification is GNU 8.1.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: C:/Qt/Tools/mingw810_64/bin/gcc.exe - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: C:/Qt/Tools/mingw810_64/bin/g++.exe - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Setting build type to 'Release' as none was specified.
-- Found Eigen: 'C:/Users/ingle/include/eigen3'
-- Could NOT find PkgConfig (missing: PKG_CONFIG_EXECUTABLE)
-- Could NOT find PkgConfig (missing: PKG_CONFIG_EXECUTABLE)
-- Could NOT find PkgConfig (missing: PKG_CONFIG_EXECUTABLE)
-- Configuring done
-- Generating done
-- Build files have been written to: C:/Users/ingle/Pangolin/build

C:\Users\ingle\Pangolin\build>

C:\Users\ingle\Pangolin\build>cmake --build .
Consolidate compiler generated dependencies of target pango_core
[  1%] Linking CXX static library libpango_core.a
[  7%] Built target pango_core
Consolidate compiler generated dependencies of target pango_image
[  8%] Linking CXX static library libpango_image.a
[ 18%] Built target pango_image
[ 18%] Embedding C:/Users/ingle/Pangolin/components/pango_opengl/shaders/colormaps.glsl.h;C:/Users/ingle/Pangolin/components/pango_opengl/shaders/font.glsl.h;C:/Users/ingle/Pangolin/components/pango_opengl/shaders/main_experiments.glsl;C:/Users/ingle/Pangolin/components/pango_opengl/shaders/main_text.glsl;C:/Users/ingle/Pangolin/components/pango_opengl/shaders/main_widgets.glsl;C:/Users/ingle/Pangolin/components/pango_opengl/shaders/matcap.glsl.h;C:/Users/ingle/Pangolin/components/pango_opengl/shaders/noise.glsl.h;C:/Users/ingle/Pangolin/components/pango_opengl/shaders/sdf.glsl.h;C:/Users/ingle/Pangolin/components/pango_opengl/shaders/utils.glsl.h into shaders.cpp
CMake Error at C:/Users/ingle/Pangolin/cmake/EmbedShaderFiles.cmake:26 (file):
  file failed to open for reading (No such file or directory):

    C:/Users/ingle/Pangolin/build/C
Call Stack (most recent call first):
  C:/Users/ingle/Pangolin/cmake/EmbedShaderFiles.cmake:61 (embed_shader_files_now)


mingw32-make.exe[2]: *** [CMakeFiles\pango_opengl.dir\build.make:86: shaders.cpp] Error 1
mingw32-make.exe[2]: *** Deleting file 'shaders.cpp'
mingw32-make.exe[1]: *** [CMakeFiles\Makefile2:641: CMakeFiles/pango_opengl.dir/all] Error 2
mingw32-make.exe: *** [Makefile:135: all] Error 2
```

--- a/cmake/EmbedShaderFiles.cmake
+++ b/cmake/EmbedShaderFiles.cmake
@@ -2,7 +2,7 @@
 # Based on http://stackoverflow.com/a/27206982
 function(embed_shader_files_now output escaped_shader_files project_prefix)
     # Undo the hack we did within embed_shader_files_rule
-    string(REGEX REPLACE ":" ";" shader_files "${escaped_shader_files}")
+    string(REGEX REPLACE "," ";" shader_files "${escaped_shader_files}")
 
     # Create empty output file
     file(WRITE ${output} "")
@@ -41,7 +41,7 @@ endfunction()
 # Sets up rule for embedding files when they are newer than the output file (or it doesn't exist yet)
 function(embed_shader_files_rule output shader_files)
     # Hack so that add_custom_command doesn't do something silly with the semi-colons.
-    string(REGEX REPLACE ";" ":" escaped_shader_files "${shader_files}")
+    string(REGEX REPLACE ";" "," escaped_shader_files "${shader_files}")
 
     add_custom_command(
         OUTPUT ${output}

